client/src/parsec/classes.ts

```diff
@@ -0,0 +1,469 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+import {
+  DeviceClaimFinalizeInfo,
+  DeviceClaimInProgress1Info,
+  DeviceClaimInProgress2Info,
+  DeviceClaimInProgress3Info,
+  libparsec,
+} from '@/plugins/libparsec';
```

Comment:
Only `libparsec` should be imported from `plugins/libparsec`, other types should be imported from `parsec/types`. This ensures that they are all exported.

Reply:
automatic imports, my bad will fix